crates/autopilot/src/run_loop.rs

```diff
@@ -308,21 +318,16 @@ impl RunLoop {
         id: domain::auction::Id,
         auction: &domain::Auction,
     ) -> Vec<Participant<'_>> {
-        let mut surplus_capturing_jit_order_owners = self
-            .cow_amm_registry
-            .amms()
-            .await
-            .into_iter()
-            .map(|cow_amm| *cow_amm.address())
-            .collect::<HashSet<_>>();
-        surplus_capturing_jit_order_owners.extend(self.surplus_capturing_jit_order_owners.clone());
-
         let request = solve::Request::new(
             id,
             auction,
             &self.market_makable_token_list.all(),
             self.solve_deadline,
-            &surplus_capturing_jit_order_owners,
+            &auction
```

Comment:
nit: if we pass down the auction into the new function, why is the extra argument still needed (can it not be taken from the auciton)?

Reply:
Really good point, I don't know how I didn't see that